harvester/aws-container-definitions.json

```diff
@@ -28,9 +23,62 @@
             "options": {
                 "awslogs-group": "/ecs/harvester",
                 "awslogs-region": "eu-central-1",
-                "awslogs-stream-prefix": "${version}",
+                "awslogs-stream-prefix": "harvester-${version}",
                 "awslogs-multiline-pattern": "^\\[?\\d\\d\\d\\d\\-\\d\\d\\-\\d\\d \\d\\d:\\d\\d:\\d\\d,\\d\\d\\d"
             }
         }
+    },
+    {
+      "name": "celery-beat-container",
+      "image": "${REPOSITORY}/harvester:${version}",
+      "cpu": 0,
```

Comment:
This thing should have a very low amount of CPU/memory. Would be a shame if it gets as much memory as UWSGI and Flower and Celery